job_definitions/clean_and_apply_db_dump.yml

```diff
@@ -19,7 +19,7 @@
           string(name: 'USERNAME', value: "clean-prod-db-dump-and-apply"),
           string(name: 'ICON', value: icon),
           string(name: 'JOB', value: "Clean and apply database dump to {{ environment }}"),
-          string(name: 'CHANNEL', value: "#dm-release"),
+          string(name: 'CHANNEL', value: "#dm-2ndline"),
```

Comment:
This is only targeted at preview/staging?

Reply:
Yeah, have updated the explanation to indicate that tests that are run exclusively on preview / staging or jobs triggered by devs/ releases should go to release.

This job will usually break due to connection issues that should be looked in to by 2ndline